### DISCUSSION TITLE
I forked and updated the EFI to latest OC version 0.7.4

### DISCUSSION THREAD

**Discussion by AlienKevin** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#43] [Show and tell]
So far on September 20th, 2021, my update is working fine for Big Sur 11.4. The updated config.plist passed OCAuxiliaryTools' syntax checks with 0 issues and all kexts are up to date. The only issue is that the tilde key for some reason is wrongly mapped to ±. I provide a simple solution in my [fork](https://github.com/AlienKevin/macOS-Dell-Latitude-E7470) and you are welcome to try out my updated EFI! I followed a guide that makes the update process mostly automatic and streamlined. I included the relevant part of the guide and a link to it in case people are interested in maintaining future updates.

Edit: Bluetooth is working now! Everything is fine. Just remember: Do not upgrade `IntelBluetoothFirmware.kext` and `IntelBluetoothInjector.kext` to the latest versions. They do not work with Dell Latitude E7470.

**Comment by adityabakare** [comment]
Great! Feel free to send a PR so that I can review and merge

**Reply by AlienKevin** [reply]
Made a PR. Just curious, how did you do the upgrades before? Did you manually edit config.plist?
